domain.c

```diff
@@ -1443,12 +1443,14 @@ int domain_check_for_local_refine(const int i, const double countlimit, const do
 {
     int j, p;
 
-    /*If there are only 8 particles within this node, we are done refining.*/
+    return 0;
```

Comment:
Please add a comment here for future reference, e.g. "Skipping local refinement because it gets horribly slow on BT-II".